### PR TITLE
Add LDAP Policy Entities command

### PR DESCRIPTION
**Pull request by donatello**
## Motivation and Context

Query LDAP policy associations.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

**Comment by donatello** [comment]
Companion to server PR https://github.com/minio/minio/pull/15908

Needs https://github.com/minio/madmin-go/pull/143

**Review by krisis — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

File: cmd/admin-idp-ldap.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ var (
 		adminIDPLdapInfoCmd,
 		adminIDPLdapEnableCmd,
 		adminIDPLdapDisableCmd,
-		// TODO: adminIDPLdapPolicyCmd,
```
Comment (vadmeste) [inline cmd/admin-idp-ldap.go:31]: It looks like command here is not correct or not needed

File: cmd/admin-idp-ldap-policy-entities.go [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+     {{.Prompt}} {{.HelpName}} play/ --policy finteam-policy --policy mlteam-policy
+  3. List all policies associated with a pair of User LDAP entities
+     {{.Prompt}} {{.HelpName}} play/ \
+              --user 'uid=bobfisher,ou=people,ou=hwengg,dc=min,dc=io' \
+              --user 'uid=fahim,ou=people,ou=swengg,dc=min,dc=io'
+  4. List all policies associated with a pair of Group LDAP entities
+     {{.Prompt}} {{.HelpName}} play/ \
+              --group 'cn=projecta,ou=groups,ou=swengg,dc=min,dc=io' \
+              --group 'cn=projectb,ou=groups,ou=swengg,dc=min,dc=io'
+  5. List all entities associated with a policy, group and user
```
Comment (vadmeste) [inline cmd/admin-idp-ldap-policy-entities.go:76]: I guess this example is little confusing and not really needed

**Review by vadmeste — COMMENTED** [review]
Few comments, LGTM otherwise

File: cmd/admin-idp-ldap-policy-entities.go (hunk 2 above)
Comment (donatello) [reply cmd/admin-idp-ldap-policy-entities.go:76]: It's just to show that you can query multiple things in a single command - we don't usually write a description of what the command does, so if we dont give examples for different usages, it's not easy to discover on their own.

**Pull request merged**